react/__tests__/index.test.tsx

```diff
@@ -635,13 +637,13 @@ describe('GA4 events', () => {
               item_name: 'Top Everyday Necessaire',
               item_variant: '20',
               price: 1600.99,
-              quantity: 1,
+              quantity: 2,
             },
           ],
           shipping: 1942.61,
           tax: 0,
           transaction_id: '1310750551387',
-          value: 3543.6,
+          value: 5144.59,
```

Comment:
[`refund`] I increased the 1600.99 item's quantity [from 1 to 2](https://github.com/vtex-apps/google-tag-manager/pull/127/files#diff-1350628461c21ae80393829f1c52fcf818a7e8c90dd7bbb0e0ef4293789b7d09L19-R19), so the total value became 3543.6 + 1600.99 = 5144.59.